src/Raygun.Blazor/Logging/RaygunLogger.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Options;
+
+namespace Raygun.Blazor.Logging
+{
+    /// <summary>
+    /// Implementation of the Raygun logger.
+    /// </summary>
+    public class RaygunLogger : IRaygunLogger
+    {
+        private readonly RaygunLogLevel _logLevel;
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="raygunSettings"></param>
+        public RaygunLogger(IOptions<RaygunSettings> raygunSettings)
+        {
+            _logLevel = raygunSettings.Value.LogLevel;
+            Info("[RaygunLogger] Logger initialized with log level: " + _logLevel);
```

Comment:
Consider adding a message here that tells developers how to shut this off. If I saw this was putting out information in the console without my explicit permission, I'd be kinda pissed.